### PR TITLE
Add SkipCertificateVerification configuration to Protocol.xsd

### PR DESCRIPTION
**Pull request by BramDevlaminck-Skyline**
## Description

The protocol.xml supports the "SkipCertificateVerification" tag to configure a default value and if the options should be disabled in the DataMiner interface or not.

## Documentation

Dataminer docs pull request that describes the same feature: https://github.com/SkylineCommunications/dataminer-docs/pull/3887

### CODE REVIEW

File: Protocol/protocol.xsd [hunk 1]
```diff
@@ -356,6 +356,7 @@ DATE        VERSION   AUTHOR    COMMENTS
 09/08/2024      Added /Protocol/Groups/Group@threadId attribute (DCP 232581) (RN 38887).
                 Added /Protocol/Threads/Thread@name attribute (DCP 232581) (RN 38887).
                 Added /Protocol/Threads/Thread@id attribute (DCP 232581) (RN 38887).
+06/11/2024      Add new tag Protocol
```
Comment (PedroDebevere) [inline Protocol/protocol.xsd:359]:
```suggestion
06/11/2024      Added /Protocol/PortSettings/SkipCertificateVerification and /Protocol/Ports/PortSettings/SkipCertificateVerification tag (DCP 195742)) (RN 40877).
```

File: Protocol/protocol.xsd [hunk 2]
```diff
@@ -14152,6 +14153,25 @@ As a bus address is not always needed, the default value can also be set to "fal
                     </xs:sequence>
                 </xs:complexType>
             </xs:element>
+            <xs:element name="SkipCertificateVerification" minOccurs="0">
```
Comment (PedroDebevere) [inline Protocol/protocol.xsd]: move element up so that the parent element are sorted alphabetically.

File: Protocol/protocol.xsd [hunk 3]
```diff
@@ -14152,6 +14153,25 @@ As a bus address is not always needed, the default value can also be set to "fal
...
+                <xs:annotation>
+                    <xs:documentation>Specifies settings related to verification process of SSL/TLS certificates.</xs:documentation>
```
Comment (PedroDebevere) [inline Protocol/protocol.xsd]: also add "Feature introduced in DataMiner a.b.c.d (RN xxxx)."

File: Protocol/protocol.xsd (hunk 2 above)
Comment (PedroDebevere) [inline Protocol/protocol.xsd]: There is another element "PortSettingsMain" (port settings for the main connection) where this item should also be added to.

**Review by BramDevlaminck-Skyline — COMMENTED** [review]
All changes were applied including the second DCP task and RN

**Review by PedroDebevere — APPROVED** [review]

**Pull request merged**